Source/WebCore/xml/XMLHttpRequestProgressEventThrottle.h

```diff
@@ -50,6 +50,7 @@ class XMLHttpRequestProgressEventThrottle {
     void updateProgress(bool isAsync, bool lengthComputable, unsigned long long loaded, unsigned long long total);
     void dispatchReadyStateChangeEvent(Event&, ProgressEventAction = DoNotFlushProgressEvent);
     void dispatchProgressEvent(const AtomString&);
+    void dispatchProgressEvent(const AtomString&, bool lengthComputable, unsigned long long loaded, unsigned long long total);
```

Comment:
Another alternative would be to name this `dispatchProgressErrorEvent` and then not pass in the false and the zeros.

Reply:
I went with your alternative proposal, I like it a little better.